lodash.js

```diff
@@ -6208,6 +6208,7 @@
      * This method is like `_.pull` except that it accepts an array of values to remove.
      *
      * **Note:** Unlike `_.difference`, this method mutates `array`.
+     * **Note:** To remove elements from an array based on a predicate, see `_.remove`.
```

Comment:
Imma call `pull` on that one. 
You hit the `pull`s eye there.
`pull` hockey I don't believe it.